spec/sections/copies.bs

```diff
@@ -308,6 +308,12 @@ dictionary GPUImageCopyExternalImage {
         region is copied into the first row of the destination region, and so on.
         The {{GPUImageCopyExternalImage/origin}} option is still relative to the top-left corner
         of the source image, increasing downward.
+
+    Note:
+    It is possible to get blank results when [=WebGL Drawing Buffer|copying from webgl/webgl2 context canvas=].
+    To avoid this, set <code>preserveDrawingBuffer<code> option in {{WebGLContextAttributes}} to <code>true<code>,
+    or issue {{GPUQueue/copyExternalImageToTexture()}} in the same event loop with
+    webgl rendering operation(ensure copy happening before webgl canvas be presented).
```

Comment:
Here's a suggestion to clarify:
- Why this happens
- The performance cost of preserveDrawingBuffer

```suggestion
    When copying from a `"webgl"`/`"webgl2"` context canvas, the
    [=WebGL Drawing Buffer=] may be not exist during certain points in the
    frame presentation cycle (after the image has been moved to the compositor
    for display). To avoid this, either:

    - Issue {{GPUQueue/copyExternalImageToTexture()}} in the same [=task=] with
        WebGL rendering operation, to ensure the copy occurs before the WebGL
        canvas is presented.
    - If not possible, set the `preserveDrawingBuffer` option in
        {{WebGLContextAttributes}} to `true`, so that the drawing buffer will
        still contain a copy of the frame contents after they've been presented.
        Note, this extra copy may have a performance cost.

```

Reply:
Thanks for suggestion. That's more clear!